src/Terminal/Run.gren

```diff
@@ -260,7 +262,7 @@ getOutputPath projectOutline =
             { outputType =
                 outputType
             , outputPath =
-                fileName
+                ".gren/" ++ fileName
```

Comment:
Feels a little iffy to hard-code this path...

Reply:
Agreed, but I think it's fine for now. In the next release or so we might add this folder to the `Compiler.Paths` module of `compiler-node`. Currently, the `.gren` folder is only used by the backend.